### PR TITLE
B 18259 

### PR DESCRIPTION
**Pull request by deandreJones**
[B-18259](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A873093)

## Summary

NFPR - Partial Delivery Out of Destination SIT Automated Testing

## Verification Steps for the Author

These are to be checked by the author.

- [ ] Have the Agility acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

- [ ] Has the branch been pulled in and checked out?
- [ ] Have the BL acceptance criteria been met for this change?
- [ ] Was the CircleCI build successful?
- [ ] Has the code been reviewed from a standards and best practices point of view?

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test
Run playwright tests for prime simulator flows
Prime simulator user :
is able submit payment request on SIT without destination SIT Out Date

### CODE REVIEW

**Review by danieljordan-caci — CHANGES_REQUESTED** [review]
Tests pass on my end, just need to add tests for functions in `pkg/factory/`:
`mto_service_item_factory.go`
`mto_shipment_factory.go`

File: pkg/factory/mto_service_item_factory.go [hunk 1]
```diff
@@ -876,6 +876,81 @@ func BuildDestSITServiceItems(db *pop.Connection, move models.Move, shipment mod
 	return []models.MTOServiceItem{ddfsit, ddasit, dddsit, ddsfsc}
 }
 
+func BuildDestSITServiceItemsNoSITDepartureDate(db *pop.Connection, move models.Move, shipment models.MTOShipment, entryDate *time.Time) models.MTOServiceItems {
```
Comment (danieljordan-caci) [inline pkg/factory/mto_service_item_factory.go:879]: ...no tests...? :(

Comment (deandreJones) [reply pkg/factory/mto_service_item_factory.go:879]: thanks for keeping me accountable... i really hate writing tests

Comment (danieljordan-caci) [reply pkg/factory/mto_service_item_factory.go:879]: Absolutely the same for me. But I refuse to let `server_test_coverage` get the best of me and that means making sure y'all write tests so it doesn't bite me in the butt later on haha

**Review by danieljordan-caci — APPROVED** [review]
Thank you for adding tests :) LGTM!

**Review by traskowskycaci — APPROVED** [review]

**Pull request merged**